cfme/infrastructure/deployment_roles.py

```diff
@@ -1,67 +1,260 @@
 """ A model of an Infrastructure Deployment roles in CFME"""
 
 from functools import partial
-from navmazing import NavigateToAttribute, NavigateToSibling
-from selenium.common.exceptions import NoSuchElementException
-
-from cfme.exceptions import ListAccordionLinkNotFound
-from cfme.fixtures import pytest_selenium as sel
-from cfme.infrastructure.provider.openstack_infra import OpenstackInfraProvider
+from navmazing import (NavigateToSibling, NavigateToAttribute)
+from widgetastic.exceptions import NoSuchElementException
+from cfme.exceptions import RoleNotFound, ItemNotFound
+from widgetastic.widget import (View, ParametrizedView)
+from widgetastic_patternfly import (BootstrapNav, Button, Dropdown, FlashMessages)
+from widgetastic_manageiq import (
+    Accordion,
+    BootstrapSelect,
+    BreadCrumb,
+    ItemsToolBarViewSelector,
+    PaginationPane,
+    SummaryTable,
+    Table,
+    Text,
+    BootstrapTreeview,
+    BaseTileIconEntity,
+    BaseQuadIconEntity,
+    BaseListEntity,
+    JSBaseEntity,
+    NonJSBaseEntity,
+    BaseEntitiesView
+)
+from widgetastic.utils import (Version, VersionPick)
+from cfme.base.ui import BaseLoggedInPage
 from utils.appliance import Navigatable
-from utils.appliance.implementations.ui import (CFMENavigateStep, navigate_to,
-                                                navigator)
-from cfme.web_ui import (listaccordion as list_acc, match_location, Quadicon,
-                         Region, toolbar as tb)
-from utils import version
-from utils.pretty import Pretty
+from utils.appliance.implementations.ui import (CFMENavigateStep, navigator, navigate_to)
+from cfme.infrastructure.provider.openstack_infra import OpenstackInfraProvider
```

Comment:
**REQURIED**
Please read more carefully style guide.

> In addition to being broken up into the three sections of standard library, third-party, and the local application, imports should be sorted alphabetically. ‘import’ lines within those sections still come before ‘from ... import’ lines

Parentheses `()` should be used only for line continuation.